domains/kryptorian.json

```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "kryptorian",
+        "email": "kryptoria@protonmail.com"
+    },
+    "record": {
+        "A": ["130.61.159.247"]
```

Comment:
I cannot reach your website
> 130.61.159.247 didn’t send any data.

Reply:
http://130.61.159.247:3000/
website is listening on port 3000